torchvision/transforms/transforms.py

```diff
@@ -165,13 +165,7 @@ def __init__(self, mean, std, inplace=False):
         self.inplace = inplace
 
     def __call__(self, tensor):
-        """
-        Args:
```

Comment:
Any particular reason why you removed this part of the docstring?

Reply:
Just to underline my point here are the three different ways `__call__` is currently documented (or not):

![image](https://user-images.githubusercontent.com/79138/66937953-9c28bf00-f040-11e9-8f0c-4309e9bc8562.png)
![image](https://user-images.githubusercontent.com/79138/66937992-ae0a6200-f040-11e9-9850-b6c41af67685.png)

Now it looks like this:
![image](https://user-images.githubusercontent.com/79138/66938159-017cb000-f041-11e9-8ec7-14407ff4fcd8.png)

Reply:
Ok with me, but I'd like to keep the input / output shape information if possible

Reply:
Should be addressed in b9a76785c382fac762827ba5a945d03cd56c3711